Lib/test/test_argparse.py

```diff
@@ -3086,20 +3105,23 @@ class TestHelpBiggerOptionals(HelpTestCase):
         0.1
         '''
 
+
 class TestShortColumns(HelpTestCase):
     '''Test extremely small number of columns.
 
     TestCase prevents "COLUMNS" from being too small in the tests themselves,
-    but we don't want any exceptions thrown in such cases. Only ugly representation.
+    but we don't want any exceptions thrown in such cases. Only ugly
+    representation.
     '''
     def setUp(self):
         env = support.EnvironmentVarGuard()
         env.set("COLUMNS", '15')
         self.addCleanup(env.__exit__)
 
-    parser_signature            = TestHelpBiggerOptionals.parser_signature
-    argument_signatures         = TestHelpBiggerOptionals.argument_signatures
-    argument_group_signatures   = TestHelpBiggerOptionals.argument_group_signatures
+    parser_signature = TestHelpBiggerOptionals.parser_signature
+    argument_signatures = TestHelpBiggerOptionals.argument_signatures
+    argument_group_signatures = TestHelpBiggerOptionals\
```

Comment:
This looked more beautiful before.

Reply:
I think that this change is good. IMO pep8 analyze should fail if we maintain the old style. Why never fail?